src/EnergyPlus/HeatingCoils.cc

```diff
@@ -2142,13 +2184,12 @@ namespace HeatingCoils {
                 // IF (FanOpMode .EQ. CycFanCycCoil) OnOffFanPartLoadFraction = 1.0d0
 
                 // Power calculation
-                HeatingCoil(CoilNum).ElecUseLoad = SpeedRatio * HSElecHeatingPower + (1.0 - SpeedRatio) * LSElecHeatingPower;
+                heatingCoil.ElecUseLoad = SpeedRatio * HSElecHeatingPower + (1.0 - SpeedRatio) * LSElecHeatingPower;
 
-                ElecHeatingCoilPower = HeatingCoil(CoilNum).ElecUseLoad;
-                HeatingCoil(CoilNum).HeatingCoilLoad = MSHPMassFlowRateHigh * (HSFullLoadOutAirEnth - InletAirEnthalpy) * SpeedRatio +
-                                                       MSHPMassFlowRateLow * (LSFullLoadOutAirEnth - InletAirEnthalpy) * (1.0 - SpeedRatio);
+                ElecHeatingCoilPower = heatingCoil.ElecUseLoad;
+                heatingCoil.HeatingCoilLoad = TotCapHS * SpeedRatio + TotCapLS * (1.0 - SpeedRatio);
```

Comment:
I am trying to find a reason for the diff in _MultiSpeedACElecFurnace.idf. That example file uses AirLoopHVAC:UnitaryHeatPump:AirToAir:MultiSpeed (no changes) with Coil:Heating:Electric:MultiStage so it must be some change in HeatingCoils that changed that answer. It looks like these are the same but I wonder if there is a rounding difference that causes the diff. Have you plotted SA and OA air flow rate and heating coil rate to see if they are very similar?

Reply:
Note that I have quickly fixed two lines of codes: https://github.com/NREL/EnergyPlus/pull/9414/files/78461457cb705a42f132d0c8037c1f66401dbba8#diff-a4469a58b3b57ae5c4916cb9efb12f0c6ef79224175d09c796a7f6c2e1d3ec33R9039
and:
https://github.com/NREL/EnergyPlus/pull/9414/files/78461457cb705a42f132d0c8037c1f66401dbba8#diff-a4469a58b3b57ae5c4916cb9efb12f0c6ef79224175d09c796a7f6c2e1d3ec33R15991

These two lines looked pretty clear to be bugs, but they might cause diffs. I will revert them and see whether that's the case.

Reply:
If you find actual bugs during development you should open a new issue and put those few lines in a seperate PR and then get that PR merged. Then when you pull develop those diffs, related to that necessary change, will be gone. And the defect documented with an issue and PR specific to that bug.